Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -186,7 +205,7 @@ namespace AzToolsFramework::Prefab
         return FocusOnOwningPrefab(focusedInstance->get().GetContainerEntityId());
     }
 
-    PrefabFocusOperationResult PrefabFocusHandler::FocusOnPrefabInstanceOwningEntityId(AZ::EntityId entityId)
+    bool PrefabFocusHandler::FindFocusedInstance(AZ::EntityId entityId, InstanceOptionalReference& outFocusedInstance)
```

Comment:
Why the change from PrefabFocusOperationResult to bool with an out argument?
If you need the Success case to return an InstanceOptionalReference, it may be easier to just change the outcome

```
using PrefabFocusOperationResult = AZ::Outcome<InstanceOptionalReference, AZStd::string>;
```

Reply:
This code has been removed.